components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/dao/RegistryPolicyDAOImpl.java

```diff
@@ -179,7 +203,7 @@ public String[] getVersions(String policyId) {
                 }
             }
         } catch (RegistryException e) {
-            LOG.error("Error while creating new version of policy", e);
```

Comment:
Was previous message wrong, and you are correcting it over here?

Reply:
Yes, this method retrieved existing versions, not creating any